Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -38,7 +38,7 @@ public override void Initialize()
         {
             base.Initialize();
 #if !OCULUSINTEGRATION_PRESENT && UNITY_EDITOR && UNITY_ANDROID
-            Debug.Log(@"Detected a potential deployment to the Oculus Quest. Be sure to download the Oculus Integration Package from the Unity Asset Store and run the Integration tool before deploying.
+            Debug.Log(@"Detected a potential deployment to the Oculus Quest. In order to use handtracking with the Oculus Quest, download the Oculus Integration Package from the Unity Asset Store and run the Integration tool before deploying.
```

Comment:
Feels like a word is missing here:

```suggestion
            Debug.Log(@"Detected a potential deployment issue for the Oculus Quest. In order to use handtracking with the Oculus Quest, download the Oculus Integration Package from the Unity Asset Store and run the Integration tool before deploying.
```